sdk/hdinsightcontainers/Azure.ResourceManager.HDInsight.Containers/tests/Azure.ResourceManager.HDInsight.Containers.Tests.csproj

```diff
@@ -8,7 +8,6 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Azure.ResourceManager.ManagedServiceIdentities" />
-    <PackageReference Include="Microsoft.Identity.Client" />
```

Comment:
WTAF?

Reply:
The main csproj (this is the tests one) has a dep on Azure.Identity so this comes transitively. The preview version in the identity package is creating version downgrade problems for this reference. Might just be an outdated reference.

Reply:
This PR should fix it - https://github.com/Azure/azure-sdk-for-net/pull/49568

We may be able to remove it also. I will determine if so when I revert the version override in the Azure.Identity package.